src/flowfield.py

```diff
@@ -129,20 +132,20 @@ def draw(self,
                       row=1, col=1
                      )
 
-        min = np.nanpercentile(streamfunction, 5)
-        max = np.nanpercentile(streamfunction, 95)
-        fig.add_trace(go.Contour(x=x_points, y=y_points, z=np.reshape(streamfunction, X.shape),
-                                 colorscale=[[0,'#000000'],[1,'#000000']],
-                                 contours=dict(start=min,
-                                               end=max,
-                                               size=(max - min) / n_contour_lines,
-                                              ),
-                                 showscale=False,
-                                 contours_coloring='lines',
-                                 hoverinfo='skip'
-                                ),
-                      row=1, col=1
-                     )
+        # min = np.nanpercentile(streamfunction, 5)
+        # max = np.nanpercentile(streamfunction, 95)
+        # fig.add_trace(go.Contour(x=x_points, y=y_points, z=np.reshape(streamfunction, X.shape),
+        #                          colorscale=[[0,'#000000'],[1,'#000000']],
+        #                          contours=dict(start=min,
+        #                                        end=max,
+        #                                        size=(max - min) / n_contour_lines,
+        #                                       ),
+        #                          showscale=False,
+        #                          contours_coloring='lines',
+        #                          hoverinfo='skip'
+        #                         ),
+        #               row=1, col=1
+        #              )
```

Comment:
This can be removed, right? The streamfunction is included below.

Reply:
Yes

Reply:
I will add some comments for it in the code